File: JobMtaani.Business.Entities/Category.cs
﻿using Core.Common.Contracts;
using Core.Common.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace JobMtaani.Business.Entities
{
    [DataContract]
    public class Category : EntityBase, IIdentifiableEntity
    {
        [DataMember]
        public int CategoryId { get; set; }
        [DataMember]
        public string CategoryName { get; set; }
        [DataMember]
        public string CategoryCName { get; set; }
        [DataMember]
        public string IconClass { get; set; }
        public object EntityId
        {
            get
            {
                return CategoryId;
            }

            set
            {
                CategoryId = (int)value;
            }
        }
    }
}

File: JobMtaani.Web/app/models/category.js
var app;
(function (app) {
    var domain;
    (function (domain) {
        var Category = (function () {
            function Category(CategoryId, CategoryName, CategoryCName) {
                this.CategoryId = CategoryId;
                this.CategoryName = CategoryName;
                this.CategoryCName = CategoryCName;
            }
            return Category;
        }());
        domain.Category = Category;
    })(domain = app.domain || (app.domain = {}));
})(app || (app = {}));
